src/listpack_malloc.h

```diff
@@ -42,4 +42,5 @@
 #define lp_malloc zmalloc
 #define lp_realloc zrealloc
 #define lp_free zfree
+#define lp_malloc_size zmalloc_size
```

Comment:
when HAVE_MALLOC_SIZE is not set, there's a difference between zmalloc_size and zmalloc_usable_size
```suggestion
#define lp_malloc_size zmalloc_usable_size
```